sql/core/src/main/scala/org/apache/spark/sql/api/python/PythonSQLUtils.scala

```diff
@@ -44,6 +48,13 @@ private[sql] object PythonSQLUtils {
     val conf = new SQLConf()
     // Force to build static SQL configurations
     StaticSQLConf
+    try {
+      val symbol = ScalaReflection.mirror.staticModule("org.apache.spark.sql.hive.HiveUtils")
+      ScalaReflection.mirror.reflectModule(symbol).instance
+    } catch {
+      case NonFatal(e) =>
+        logError("Cannot generated sql configurations from hive module", e)
```

Comment:
How about using `logWarning` here? I looked around the existing codes using `logError` and I feel they are more critical errors.

Reply:
OK